docs/Using-Fleet/fleetctl-CLI.md

```diff
@@ -189,6 +190,99 @@ spec:
   query: select * from processes
 ```
 
+## Using fleetctl with an API-only user
+
+Now that `fleetctl` and the Fleet server is configured, it can be helpful to create an API-only user to use when running automated workflows. An API-only user can be given a role based on the abilities it needs. The default access level is `Observer`. For more information on roles, see the [user permissions documentation](./Permissions.md#user-permissions).
+
+### Create an API-only user
+
+To create your new API-only user using `fleetctl user create` and picking a name, email, and password and setting the `api-only` flag to `true`:
```

Comment:
This sentence is not a complete thought, and the part after the colon doesn't make it one either. I'm not sure what needs to be added. If the action is typing the below into the terminal, let's include that. Please reword to make this a complete sentence/thought.

Reply:
@ksatter

Reply:
That really wasn't a sentence at all, thanks! Changes committed.